lib/trento/activity_logging/phoenix_conn_parser.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Trento.ActivityLogging.Logger.PhoenixConnParser do
```

Comment:
Could this potentially be a Protocol?

Reply:
I did not think about that, yet. I cannot firmly say yes or no at the moment.
I guess when we introduce the Commanded related activity logging we might discover what it needs to be.